src/common/token_bucket.c

```diff
@@ -20,6 +21,51 @@
 #include "token_bucket.h"
 #include "util_bug.h"
 
+/**
+ *Set the <b>rate</b> and <b>burst</b> value in a token_bucket_cfg.
+ *
```

Comment:
Whitespace missing.

Reply:
Fixed in 1be024798e2e2bac499acf8106eb215e3c4c130b